AutomatedTesting/Gem/PythonTests/scripting/ScriptEvents_AllParamDatatypes_CreationSuccess.py

```diff
@@ -66,23 +63,40 @@ class TestScriptEvents_AllParamDatatypes_CreationSuccess():
 
     def __init__(self):
 
-        editor_window = None
-        asset_editor = None
-        asset_editor_widget = None
-        asset_editor_row_container = None
-        asset_editor_menu_bar = None
-        script_canvas = None
-        node_palette = None
-        node_tree_view = None
-        node_tree_search_frame = None
-        node_tree_search_box = None
+        self.editor_main_window = None
+        self.asset_editor = None
+        self.asset_editor_widget = None
+        self.asset_editor_row_container = None
+        self.asset_editor_menu_bar = None
+        self.sc_editor = None
+        self.sc_editor_main_window = None
+        self.node_palette = None
+        self.node_tree_view = None
+        self.node_tree_search_frame = None
+        self.node_tree_search_box = None
 
     def verify_added_params(self):
         for index in range(N_VAR_TYPES):
             if self.asset_editor_row_container.findChild(QtWidgets.QFrame, f"[{index}]") is None:
                 return False
         return True
 
+    def set_name_field_names(self):
+        index = 0
+        name_fields = tools.get_script_event_parameter_name_text(self)
+        for name_field in name_fields:
+            if name_field is not None and name_field.text() == PARAMETER_NAME:
+                name_field.setText(f"param_{index}")
+                index += 1
+
+    def set_type_field_types(self):
```

Comment:
Add Doc Strings to local helper functions

Reply:
added